src/pages/lib/ssr/q/platform/[platform].mdx

```diff
@@ -1,8 +0,0 @@
-export const meta = {
```

Comment:
We're deleting this page, but also linking to it from the v5 directory structure. We should pick one or the other so we don't have a broken link.

Reply:
Oh no this should not be included in this PR.